python/paddle/distributed/auto_parallel/converter.py

```diff
@@ -0,0 +1,449 @@
+                format(str(type(params_dict))))
+        self._params_dict = params_dict
+
+    @pre_strategy.setter
+    def pre_strategy(self, pre_strategy):
+        if not pre_strategy:
+            raise ValueError("'pre_strategy' is None, "
+                             "there are not parameter in pre process.")
+        if not isinstance(pre_strategy, dict):
+            raise TypeError("The type of 'pre_strategy' should be 'dict', "
+                            "but got '{}'.".format(str(type(pre_strategy))))
+        self._pre_strategy = pre_strategy
+
+    @cur_strategy.setter
+    def cur_strategy(self, cur_strategy):
+        if not cur_strategy:
+            warnings.warn("'cur_strategy' is None, "
+                          "there are not parameter in cur process")
+        if not isinstance(cur_strategy, dict):
+            raise TypeError("The type of 'cur_strategy' should be 'dict', "
+                            "but got '{}'.".format(str(type(cur_strategy))))
+        self._cur_strategy = cur_strategy
+
+    def convert(self, strict=True):
+        """
+        convert parameters
+
+        Returns:
+            converted parameters(dict)
+
+        Examples:
+            .. code-block:: python
+
+                import numpy as np
+                complete_param = np.arange(4).reshape([2, 2])
+                partitial_param = np.split(complete_param, 2, axis=0)
+                name = "param_0"
+                param_dict = {name: partitial_param}
+                strategy_1 = {
+                    name: {
+                        "process_shape": [2],
+                        "process_group": [0, 1],
+                        "dims_mapping": [0, -1]
+                    }
+                }
+                strategy_2 = {
+                    name: {
+                        "process_shape": [2],
+                        "process_group": [0, 1],
+                        "dims_mapping": [-1, -1]
+                    }
+                }
+                converter = Converter(param_dict, strategy_1, strategy_2)
+                result = converter.convert()
+                # the result's value is equal to `complete_param`
+        """
+        params_dict = {}
+        # the name which is in cur_process but not in pre_process
+        param_not_in_pre = []
+        # the name which is in pre_process but not in cur_process
+        param_not_in_cur = []
+        # the name which is in strategy but not in ckpt files
+        param_not_in_ckpt = []
+        self._logger.info("Start to convert parameters.")
+        for param_name in self._cur_strategy:
+            if param_name not in self._pre_strategy:
+                param_not_in_pre.append(param_name)
+                continue
+            if param_name not in self._params_dict:
+                param_not_in_ckpt.append(param_name)
+                continue
+
+            param_list = self._params_dict[param_name]
```

Comment:
"param_xxx" --> "tesnor_xxx" a more proper and meaningful naming

Reply:
done.